packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -877,6 +877,11 @@ class FlutterError extends Error with DiagnosticableTreeMixin implements Asserti
   ///
   /// Do not call [onError] directly, instead, call [reportError], which
   /// forwards to [onError] if it is not null.
+  ///
+  /// See also:
+  ///
+  ///  * <https://flutter.dev/docs/testing/errors>, more information about error
+  ///    handling in Flutter.
```

Comment:
This is linked above in the `FlutterError` doc comment, but I think this can be also a useful place to have this link